bin/travis-ci/check_tag.sh

```diff
@@ -25,32 +25,50 @@ sudo chmod 775 "${shtdlib_local_path}"
 source "${shtdlib_local_path}"
 color_echo green "shtdlib.sh installed successfully"
 
+version_pattern='v\d+\.\d+\.\d+(?:qa)?'
+version_pattern_line="^${version_pattern}$"
```

Comment:
Do we need to split this regex?

Reply:
Yeah... for later on

Reply:
The CHANGELOG has a different check because its not a full line

Reply:
ah ok